helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedInstanceCapacity.java

```diff
@@ -54,8 +55,15 @@ public WagedInstanceCapacity(ResourceControllerDataProvider clusterData) {
       return;
     }
     for (InstanceConfig instanceConfig : clusterData.getInstanceConfigMap().values()) {
-      Map<String, Integer> instanceCapacity = WagedValidationUtil.validateAndGetInstanceCapacity(clusterConfig, instanceConfig);
-      _instanceCapacityMap.put(instanceConfig.getInstanceName(), instanceCapacity);
+      Map<String, Integer> instanceCapacity = null;
+      try {
+        instanceCapacity = WagedValidationUtil.validateAndGetInstanceCapacity(clusterConfig, instanceConfig);
+      } catch (HelixException ex) {
+        // We don't want to throw exception here, it would be OK if no resource is using Waged.
+        // Waged rebalancer will fail in later pipeline stage only for waged resource. So it won't block other resources.
+        LOG.error("Failed to initialize instance capacity map. Instance capacity map is not property set up", ex);
```

Comment:
Let's remove this. Anyway in WAGED rebalanced it will have a log, which is duplicated.